strongbox-db-server/src/main/java/org/strongbox/db/server/janusgraph/TransactionalVertexIDAssigner.java

```diff
@@ -0,0 +1,157 @@
+package org.strongbox.db.server.janusgraph;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import org.janusgraph.core.VertexLabel;
+import org.janusgraph.diskstorage.BackendException;
+import org.janusgraph.diskstorage.IDAuthority;
+import org.janusgraph.diskstorage.IDBlock;
+import org.janusgraph.diskstorage.configuration.Configuration;
+import org.janusgraph.diskstorage.keycolumnvalue.KeyRange;
+import org.janusgraph.diskstorage.keycolumnvalue.StoreFeatures;
+import org.janusgraph.graphdb.database.idassigner.IDBlockSizer;
+import org.janusgraph.graphdb.database.idassigner.VertexIDAssigner;
+import org.janusgraph.graphdb.idmanagement.IDManager;
+import org.janusgraph.graphdb.internal.InternalRelation;
+import org.janusgraph.graphdb.internal.InternalVertex;
+
+/**
+ * @author sbespalov
+ */
+public class TransactionalVertexIDAssigner extends VertexIDAssigner
+{
+
+    public static final String QUEUE_DEFAUT = "default";
```

Comment:
Typo - not sure if this is used anywhere else down the line so not going to commit it.

```suggestion
    public static final String QUEUE_DEFAULT = "default";
```

Reply:
fixed